### PR TITLE
Adding full-post styles for Discover's intro class

### PR DESCRIPTION
**Pull request by kjellr**
Adding an “intro” style to Discover’s full post views in the Reader.
This is meant to mimic intro text hierarchy styles used on discover.wordpress.com. Only applies to the full post view for Discover (blog id 53424024). This class visually matches the current .reader__full-post-content h3 style.

We'd previously been using h tags to emulate this style in the Reader, but that's not a semantic approach. 

You can test this branch by viewing the full post view for two recent Discover features. Both these posts use <p class="intro"> tags for their first paragraph.

Example Post: https://discover.wordpress.com/2016/01/21/fennabee/
Full Post View: http://calypso.localhost:3000/read/post/id/53424024/9037

Example Post 2: https://discover.wordpress.com/2016/01/19/can-comedy-be-comfortable/
Full Post View 2: http://calypso.localhost:3000/read/post/id/53424024/8589

cc @shaunandrews

### CODE REVIEW

**Comment by kjellr** [comment]
Screenshots: 
screen shot 2016-01-22 at 11 55 07 am

screen shot 2016-01-22 at 11 57 11 am

**Comment by blowery** [comment]
Can we get the dropcap too? :D

**Comment by kjellr** [comment]
@blowery Yep! 

Screenshots: 
screen shot 2016-01-22 at 12 25 13 pm
screen shot 2016-01-22 at 12 24 38 pm

File: client/reader/full-post/_style.scss [hunk 1]
```diff
@@ -316,3 +316,29 @@
...
+			font-weight: 400;
+			line-height: 0.5em;
+			margin-top: 0.4em;
+			margin-right: 0.15em;
+		}
+
+		// Firefox-only position fix
+		@-moz-document url-prefix() {
+			&:first-child:first-letter {
+				margin-top: 0.25em;
```
Comment (blowery) [inline client/reader/full-post/_style.scss]: looks better to me at 0.2em? .25 seems too far down.

**Comment by blowery** [comment]
here's what i'm seeing in firefox 43.0.4

screenshot

**Comment by blowery** [comment]
All other browsers check out. :+1:

**Comment by kjellr** [comment]
Good call. 12703bd has: 
- Better alignment in FF
- A slightly larger text size so we can line up with both the top of the first line, and the baseline of the second.
- Pixel margin measurements instead of ems, to match the rest of the stylesheet.

screen shot 2016-01-22 at 1 34 44 pm

**Comment by blowery** [comment]
Nice! :ship:

**Comment by shaunandrews** [comment]
Super awesome!

**Pull request merged**